composable/chain.json

```diff
@@ -6,8 +6,8 @@
   "pretty_name": "Composable",
   "status": "live",
   "network_type": "mainnet",
-  "bech32_prefix": "banksy",
-  "daemon_name": "banksyd",
+  "bech32_prefix": "centauri",
```

Comment:
I don't know how it slipped by me earlier, but the bech32 prefix aught to be registered to SLIP173 https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Did a PR for it https://github.com/satoshilabs/slips/pull/1568

Composable also decided last minute to change the version for centaurid. Will update my PR in a second.